dhis-2/dhis-api/src/main/java/org/hisp/dhis/scheduling/parameters/SmsInboundProcessingJobParameters.java

```diff
@@ -25,16 +25,22 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package org.hisp.dhis.sms;
+package org.hisp.dhis.scheduling.parameters;
 
-import org.hisp.dhis.sms.incoming.IncomingSms;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hisp.dhis.scheduling.JobParameters;
 
-public interface MessageQueue {
-  void put(IncomingSms message);
+@Getter
+@Setter
+@NoArgsConstructor
+public class SmsInboundProcessingJobParameters implements JobParameters {
+  @JsonProperty(required = true)
+  private long smsId;
```

Comment:
This should be UID since job parameters are observable in the API. Then also add `@OpenApi.Property({UID, ?})` - have a look at other parameter classes that have ID parameters. `?` because I am not sure what the object type for SMS is :D